signup.css

```diff
@@ -0,0 +1,60 @@
+.SignUp-banner-heading{
+    background:url(./images/banner.jpg) no-repeat;
+    background-size: cover;
+    background-position: center;
+    text-align: center;
+    padding-top: 12rem;
+    padding-bottom: 8rem;
+}
+  
+.SignUp-banner-heading h1{
+      color:var(--black);
+      font-size: 4rem;
+}
+
+.SignUp-banner-heading h1 span{
+    color: var(--orange);
+}
```

Comment:
be consistent with naming. use kebab case only, don't mix camel case

Reply:
ok